app/_hub/kong-inc/acme/_index.md

```diff
@@ -288,6 +289,47 @@ schedules a renewal in the background and returns immediately.
 curl http://localhost:8001/acme -XPATCH
 ```
 
+## Monitoring and debugging
+
+The ACME plugin exposes several endpoints through Admin API that can be used for
+debugging and monitoring certificate creation and renewal.
+
+- **POST /acme**: start applying or renewing certificate and return the result; available parameter:
+  - **host**: the domain to create certificate
+  - **test_http_challenge_flow**: when set, only check for configuration sanity.
```

Comment:
I suggest using the formatting that we use for requests elsewhere (here's another plugin documented in the same way: https://docs.konghq.com/hub/kong-inc/proxy-cache-advanced/):

```suggestion
### Apply certificate

**Endpoint**

<div class="endpoint post">/acme</div>
Applies or renews the certificate and returns the result.

**Request body**

Attribute | Description
---:| ---
`host`<br>*required* | The domain where to create the certificate.
`test_http_challenge_flow`<br>*optional* | When set, only checks if the configuration is valid. Does not apply the certificate.
```